align/schema/instance.py

```diff
@@ -10,8 +10,7 @@ class Instance(types.BaseModel):
     name: str
     pins : Dict[str, str]
     parameters : Optional[Dict[str, str]]
-    generator: str #Handles different sized instantiation of same subcircuit to same generator
-    abstract_name: Optional[str] # Added during primitive generator, in case no primitive generator found = generator
+    abstract_name: Optional[str] # unique name based on model and parameters
```

Comment:
@kkunal1408 , iif it is a trivial change, should we stick with `abstract_template_name` for now? In a future PR, we can rename as `abstract_name` thru the ALIGN flow.

Reply:
it was previously also abstract_name and it was not modified. Let me know if you want me to change abstract_template_name to abstract_name everywhere or vice versa. I think it's a similar amount of effort.

Reply:
Sorry, abstract_name is local to compiler and primitive generator, while abstract_template is local to primitive_generator and PnR. I won't like to touch the PNR so I can only do the abstract_name to abstract_template_name change in PNR and primitive_generator early part.

Reply:
Let's take care of this in a future PR.